Khan/wonder-blocks
# Ensure proto packages are stripped from release docs

## Test Instructions
1. `yarn start`
1. `yarn start:prod`
1. Compare

Packages marked `private: true` in their `package.json` should be omitted from the prod variant.
